tool-execution/pmd_setup.sh

```diff
@@ -0,0 +1,41 @@
+## This script sets up PMD on an Ubuntu 20.04 or MacOS 11 machine.
```

Comment:
Please run the [shellcheck](https://www.shellcheck.net/) tool on this script (it fails) and fix the resulting errors.

It would also be great to setup shellcheck in CI for this repo if we're going to be writing new shell scripts.